modules/javafx.graphics/src/main/java/javafx/application/Application.java

```diff
@@ -32,6 +32,7 @@
 import javafx.application.Preloader.PreloaderNotification;
 import javafx.css.Stylesheet;
 import javafx.scene.Scene;
+import javafx.scene.paint.Color;
```

Comment:
strictly speaking, this file should be unchanged.

Reply:
Agreed. Now that the user preferences API has been split into a follow-on enhancement, which was a very good idea, there is no reason to change this file.

Reply:
Fixed.